rr_viz/src/ross/estop/emergency_stop.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python
+from PyQt5.QtWidgets import QVBoxLayout, QWidget, QLabel, QPushButton, QHBoxLayout
+from PyQt5.QtGui import QFont
+from PyQt5.QtCore import Qt
+
+import rospy
+from std_srvs.srv import SetBool, SetBoolRequest
+from std_msgs.msg import Bool
+
+class EmergencyStop(QWidget):
+    def __init__(self, parent):
+        super(QWidget, self).__init__(parent)
+
+        self.v_layout = QVBoxLayout()
+
+        # Title
```

Comment:
I would start at least putting the layout initialization in its own function, so that it doesn't 'pollute' other logic in the code.

If creating a functions that purely deal with layout I would prefix them with layout_*. In this case perhaps something like layout_init(self).

Reply:
done :)